### PR TITLE
Added URL API to Foundation.Process

### PR DESCRIPTION
**Pull request by pvieito**
Refs #1461

### CODE REVIEW

**Comment by pvieito** [comment]
I've also added a small fix to Locale static properties that produces crashes on Linux by bridging String to NSString.

**Comment by pushkarnk** [comment]
@swift-ci please test

**Comment by alblue** [comment]
@swift-ci please test

**Comment by pvieito** [comment]
@swift-ci please test

File: Foundation/NSLocale.swift [hunk 1]
```diff
@@ -84,11 +84,11 @@ open class NSLocale: NSObject, NSCopying, NSSecureCoding {
 }
 
 extension NSLocale {
-    open class var current: Locale {
+    open static var current: Locale {
```
Comment (millenomi) [inline Foundation/NSLocale.swift]: Why `static`? This doesn't match the Darwin signature:

```swift
class var current: Locale { get }
```

File: Foundation/NSLocale.swift [hunk 2]
```diff
@@ -84,11 +84,11 @@ open class NSLocale: NSObject, NSCopying, NSSecureCoding {
...
         return CFLocaleCopyCurrent()._swiftObject
     }
     
-    open class var system: Locale {
+    open static var system: Locale {
```
Comment (millenomi) [inline Foundation/NSLocale.swift]: As above for all `static` members. Please revert?

**Comment by pvieito** [comment]
Ups, sorry, didn't think about that. I will resend two separated pull requests.

**Pull request closed without merging**